docs/source/user-guide/install/macos.rst

```diff
@@ -55,29 +57,23 @@ the bash installer. The following arguments are supported:
 * ``-p``: Installation prefix/path.
 * ``-f``: Force installation even if prefix ``-p`` already exists.
 
-EXAMPLE:
+**Example**
```

Comment:
You could make these examples sub-headlines

Reply:
I thought about doing that, but then it populates the right sidebar menu. I might be able to restrict how deep this menu goes somehow though.

Reply:
I _think_ this is the setting, but it is global. https://pydata-sphinx-theme.readthedocs.io/en/stable/user_guide/page-toc.html#show-more-levels-of-the-in-page-toc-by-default